connect/api/src/main/java/org/apache/kafka/connect/connector/Connector.java

```diff
@@ -43,6 +46,7 @@
 public abstract class Connector {
 
     protected ConnectorContext context;
+    protected ConfigDef configDef;
```

Comment:
This should be removed since we changed the implementation of `validate` and the implementations of `config()` in subclasses can be simplified to use a `private static final ConfigDef` of their own.